### ISSUE TITLE
Versioning on LuaRocks (and in general)

### ISSUE THREAD

**Issue by pranas** [excessive/cpml#22]
Hey guys

Thanks for releasing the rock (#18)! We used it and it was great... but the update on Dec 14 broke our app. Could you start versioning as well? If you don't have time we could help make releases from time to time.

**Comment by karai17** [comment]
LuaRocks hasn't been updated in 110 days so I'm not sure how your code broke.

**Comment by pranas** [comment]
It would appear so but I believe ``scm`` versions are like nightlies and follow trunk/master, see [wiki](https://github.com/luarocks/luarocks/wiki/creating-a-rock#method-1-using-a-repository-such-as-github). Sadly release dates don't get updated on LuaRocks website.

**Comment by shakesoda** [comment]
@karai17 yeah, scm versions are nightlies. luarocks version isn't locked to that old one.

@pranas sorry for breaking your app, some help with releases would be much appreciated. API should be *mostly* stable right now with exception to mat4 (I reverted the API style change because it broke our skeletal animation code, and I haven't made time to debug it). I made sure to tag the last revision before the API break, if it helps: https://github.com/excessive/cpml/tree/v0.9.0. Is there a good way to release a rock of just a specific tag?

note that we actually do have a _VERSION field in the module (https://github.com/excessive/cpml/blob/master/init.lua#L40), which gets changed when we break API significantly.
